data-manipulation/hashing/murmur/hash-data-using-murmur3.yml

```diff
@@ -28,3 +28,25 @@ rule:
         - and:
           - number: 0x87c37b91114253d5 = 64-bit c1 for 128-bit hash
           - number: 0x4cf5ad432745937f = 64-bit c2 for 128-bit hash
+        - basic block:
+          - and:
+            - count(mnemonic(shr)): 3
+            - number: 16
+            - number: 13
+            - optional:
+              - count(characteristic(nzxor)): 3 or more

```

Comment:
Here we need `3 or more` because there are more `nzxor` in murmur which may be matched as well. I am looking to a sample whose murmur3 implementation looks like the Wikipedia one and capa matches 4.